roadside-forms-frontend/frontend_web_app/src/utils/dateTime.js

```diff
@@ -0,0 +1,9 @@
+export const getCurrentDateTime = () => {
```

Comment:
Created this common function to get the date time and using it in useEffect to get latest date and time every minute. I think it can be used anywhere in the application.